packages/tupaia-web/src/features/Map/MapOverlaySelector/DesktopMapOverlaySelector.tsx

```diff
@@ -146,47 +164,59 @@ export const DesktopMapOverlaySelector = ({
 }: DesktopMapOverlaySelectorProps) => {
   const { projectCode, entityCode } = useParams();
   const { hasMapOverlays } = useMapOverlays(projectCode, entityCode);
-  const { period } = useActiveMapOverlayReport();
-
+  const { data: mapOverlayData } = useMapOverlayReport();
+  const [mapModalOpen, setMapModalOpen] = useState(false);
+  const handleOpen = () => {
+    setMapModalOpen(true);
+  };
   return (
-    <Wrapper>
-      <Header>
-        <Heading>Map Overlays</Heading>
-      </Header>
-      <Container>
-        <TitleWrapper>
-          <MapOverlaySelectorTitle />
-          <MapOverlayDatePicker />
-        </TitleWrapper>
-        {hasMapOverlays && (
-          <OverlayLibraryAccordion
-            expanded={overlayLibraryOpen}
-            onChange={toggleOverlayLibrary}
-            square
-          >
-            <OverlayLibraryHeader
-              expandIcon={<ArrowDropDown />}
-              aria-controls="overlay-library-content"
-              id="overlay-library-header"
+    <>
+      {mapModalOpen ? <MapTableModal setMapModalOpen={setMapModalOpen} /> : null}
```

Comment:
I think it might be nice to pass in `closeModal={() => setModalOpen(false)}` instead of `setMapModalOpen`